gigascience/gigadb-website
# fix table column headings overlap with caret

# Pull request for issue: #1856

This is a pull request for the following functionalities:

* Extend right padding to add more spacing between caret and table header text

![image](https://github.com/gigascience/gigadb-website/assets/143437854/aaee53f3-a089-4562-9acc-961d882e8470)


## How to test?

* Navigate to e.g. http://gigadb.gigasciencejournal.com/dataset/100094
* Resize screen to check that caret and table headers never overlap
